### ISSUE TITLE
All steps in feature files are marked as unmatched and a NullPointerException

### ISSUE THREAD

**Issue by Doogiemuc** [cucumber/cucumber-eclipse#151]
I really love cucumber! And Cucumber-Eclipse plugin is great. But after the latest update (march 28th  0.0.13.201603281006) I have a problem: 

All my steps in every feature file are marked as "unmatched". It seems like the plugin cannot find my (java) stepdefs. How do I configure the path or java package where it looks for them?

In one feature file the cucumber-eclipse editor even crashes with a NullPointerException, when I try to opoen the file.   It is a rather elaborate feature file. Here is the stack trace:

```
java.lang.NullPointerException
    at cucumber.eclipse.editor.editors.GherkinErrorMarker.markUnmatchedStep(GherkinErrorMarker.java:206)
    at cucumber.eclipse.editor.editors.GherkinErrorMarker.step(GherkinErrorMarker.java:144)
    at gherkin.formatter.model.Step.replay(Step.java:68)
    at gherkin.formatter.model.Step$Builder.replay(Step.java:41)
    at gherkin.parser.FormatterListener.replayStepsOrExamples(FormatterListener.java:151)
    at gherkin.parser.FormatterListener.step(FormatterListener.java:120)
    at gherkin.parser.Parser.step(Parser.java:128)
    at gherkin.lexer.En.scan(En.java:788)
    at gherkin.lexer.I18nLexer.scan(I18nLexer.java:30)
    at gherkin.parser.Parser.parse(Parser.java:58)
    at cucumber.eclipse.editor.editors.Editor.validateAndMark(Editor.java:202)
    at cucumber.eclipse.editor.editors.Editor.doSetInput(Editor.java:149)
    at org.eclipse.ui.texteditor.AbstractTextEditor$19.run(AbstractTextEditor.java:3220)
    at org.eclipse.jface.operation.ModalContext.runInCurrentThread(ModalContext.java:463)
    at org.eclipse.jface.operation.ModalContext.run(ModalContext.java:371)
    at org.eclipse.ui.internal.WorkbenchWindow$14.run(WorkbenchWindow.java:2181)
    at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:70)
```

Some more information of my project setup:
- My features are in `src/test/feature/*.feature`
- My java step defs ("glue code") are in `src/test/java/com.company.test.cucumber`
- I run my cucumber tests via JUnit. Here is my JUnit file:

```
@RunWith(Cucumber.class)
@CucumberOptions(plugin = {"pretty"}, features = {"src/test/feature"}, glue = "com/bmw/test/cucumber")
public class RunAllCucumberTests {
}
```

I am not quite sure if that CucumberOption  `glue= "...."` is correct like that.

**Comment by ilanpillemer** [comment]
Could you provide maybe a simple example that allows this to be recreated?

**Comment by Doogiemuc** [comment]
**SOLUTION:**   It was an line that only contained the word   "And"

**Comment by ilanpillemer** [comment]
@Doogiemuc  Can we close this issue, there already is an open issue for step definitions in class files instead of in project?

**Comment by ilanpillemer** [comment]
Leaving open as this is a bug obviously when there is only an 'And'

**Comment by hdpe** [comment]
This was fixed in 6495170281732c80a2d61bc3e4fefcf558f4fc38
